cluster/examples/kubernetes/ceph/pool.yaml

```
@@ -18,6 +18,8 @@ spec:
     # Disallow setting pool with replica 1, this could lead to data loss without recovery.
     # Make sure you're *ABSOLUTELY CERTAIN* that is what you want
     requireSafeReplicaSize: true
+    # The number for replica per failure domain, the value must be lower than the number of replica and a multiple of 2
```

It doesn't have to be a multiple of 2, right? But I don't know why they would choose anything else.
```suggestion
    # The number for replicas per failure domain, the value must be a divisor of the replica count. If specified, the most common value is 2 for stretch clusters, where the replica count would be 4.
```